lib/sass.dart

```diff
@@ -15,12 +16,12 @@ import 'src/visitor/serialize.dart';
 /// If [color] is `true`, this will use terminal colors in warnings.
 ///
 /// Throws a [SassException] if conversion fails.
-String render(String path, {bool color: false}) {
+Future<String> render(String path, {bool color: false}) async {
```

Comment:
if `render` function will take `SyncPackageResolver` as parameter, how should I initialize the passed variable?